### PR TITLE
Scroll the graph selection pane with the performance graphs

### PR DESCRIPTION
**Pull request by ronawho**
Previously the graph selection pane (suite selection, individual graph
selection, etc) stayed stationary and you had to scroll back up to the top of
the page to select different graphs. Now it scrolls as you scroll the page. In
order to do that the individual graph selection list had to be put inside a new
pane so it can scroll if there's more graphs than can fit on the current page.

This adds logic to do the scrolling and automatically adjust the height of the
graph list on resizes and set the initial position of the graphPane based on
the width of the graphSelectionPane

### CODE REVIEW

**Comment by ronawho** [comment]
@bradcray do you mind giving this a quick look? You don't have to review the code changes, just see if http://ejrgraphtest.sourceforge.net/perf/scrollbarTest/ looks right to you.

**Comment by bradcray** [comment]
This looks OK to me.  When scrolling the main window, the pane with 
checkboxes sortof jumps around wildly before settling down (in Chrome on 
Windows), but I'd rather have it this way than the old way.

-Brad

**Comment by ronawho** [comment]
Huh that's odd, that doesn't happen with chrome on my machine (but I also see it when vpn'd into a windows box.) Let me take a quick look and see if there's any easy fix

**Comment by ronawho** [comment]
ok, check it out now. It should be better

**Comment by bradcray** [comment]
Yep much better.  The only thing that seems weird to me now (but again, I wouldn't put it in the way of committing) is that if I scroll (via mouse wheel, say) past the top or bottom of the checkbox sidebar, the main window scrolls.  Not sure if this is bug or feature.

**Comment by ronawho** [comment]
Cool. 

tl;dr: The scrolling thing isn't a "bug" but it can be (easily) changed in the future if it gets annoying.

If you put too many things on a page there is a default page scroller that gets activated. We rely on that for the graphs/legends.

When you're inside the graphlist and scrolling it intercepts the page scroll listener. Once the graphlist is scrolled to the bottom and you keep scrolling down it is no longer intercepting the scroll action (since it isn't actually scrolling) so the page intercepts them. 

If it gets annoying we can put the graphs inside a new div that has it's own scrolling instead of using the default page scroller.
